vmdk_plugin/drivers/photon/photon_driver.go

```diff
@@ -560,7 +560,13 @@ func (d *VolumeDriver) Create(r volume.Request) volume.Response {
 func (d *VolumeDriver) Remove(r volume.Request) volume.Response {
 	log.WithFields(log.Fields{"name": r.Name}).Info("Removing volume ")
 
-	// Docker is supposed to block 'remove' command if the volume is used. Verify.
+	if d.refCounts.GetInitSuccess() != true {
```

Comment:
If it remains false then no volume can be unmounted? Its as good as all volumes are attached and remain so? Could add a comment here to briefly explain this.

Please add a test to mount a volume and then on unmount it gets this error, so the volume is now left attached to the VM till the VM is powered off.

Possibly add the volumes (name) to a list and when the refcounting is done initializing, process the list to unmount volumes if needed.